PICMI_Python/particles.py

```diff
@@ -282,6 +282,77 @@ def __init__(self, density,
         self.handle_init(kw)
 
 
+class PICMI_FoilDistribution(_ClassWithInit):
+    """
+    describes a foil with optional exponential pre-plasma ramps (front lower y/back higher y)
+
+    density: float
+        Physical number density [m^-3]
+
+    y_front : float
+        y-postion of front surface of foil [m]
+
+    thickness : float > 0
```

Comment:
For coding standards, no space before the colon. Also, capitalize the start of the documentation.
```suggestion
    thickness: float > 0
```